README.md

```diff
@@ -0,0 +1,19 @@
+WORKFLOW CA- social media client
+
+Prerequisites Git Node JS VS-code/other compiler

```

Comment:
This file could use some formatting. I recommend reading about markdown formatting.